main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"dynamic-buildkite-template/generator"
+	"os"
+)
+
+func main() {
+	var trivyPlugin = "v1.18.0"
+	var shellPlugin = "v1.3.0"
+
+	generator.GenerateTrivyStep(trivyPlugin, shellPlugin, os.Stdout)
```

Comment:
Now that `GenerateTrivyStep` returns an error (as opposed to no return values), you need to check if the returned error is non-nil. See my other comment for an example :)

Reply:
fixed